tools/release/publish_release.sh

```diff
@@ -767,14 +750,14 @@ if [ "$cherry_pick_resolved" = "false" ]; then
 
     commits=""
 
-    if [[ "$main_release" == "false" ]]; then
+    if [[ "$minor" == "false" ]]; then
```

Comment:
we need to leave this split since I just did the version where I needed minor and cherry-picks.

Reply:
Good point. I've added a new option to allow minors with cherry picks.